lambda/latest_image_tracker/image-builder-lambda-update-ssm.py

```diff
@@ -1,80 +1,72 @@
 import json
 import boto3
-import logging 
+import logging
 
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 
-ssm_client = boto3.client('ssm')
-ssm_parameter_name = '/ec2-imagebuilder/latest'
+# ssm_client = boto3.client('ssm')
+ssm_parameter_name = "/ec2-imagebuilder/latest"
```

Comment:
Given we have a mix of both ```'``` and ```"``` across the repository, can we leave these as is in this sample? (otherwise we'll end up just flipping these back and forth depending on who files pull requests!)